docs/source/guides/user/chapters/dependencies.rst

```diff
@@ -196,6 +196,33 @@ effect on the spawner.
  * `uri`: the image reference, in any format supported by ``podman
    pull`` itself.
 
+VM Image
+++++++++
+
+Support downloading virtual machine images ahead of test execution time.
+This allows tests to have their required VM images downloaded and cached
+before the test execution begins, preventing timeout issues during the
+actual test run.
+
+ * `type`: `vmimage`
+ * `provider`: the VM image provider (e.g., 'Fedora')
+ * `version`: version of the image
+ * `arch`: architecture of the image
+
+Following is an example of a test using the VM Image dependency::
+
+    from avocado import Test
```

Comment:
Instead of hardcoded example, I would use link to `examples/tests/dependency_vmimage.py`. It will help us with maintainability of docs.